packages/recomponents/src/setup-test.js

```diff
@@ -0,0 +1,10 @@
+import Vue from 'vue';
+
+import * as directives from './directives';
+import kebabCase from './common/helpers/kebab-case';
+
+// Register directives in the global vue instance,
+// as they are used in most of our components
+Object.keys(directives).forEach((key) => {
+    Vue.directive(kebabCase(key.substr(1)), directives[key]);
+});
```

Comment:
Maybe it's better to use `createLocalVue` and pass it to tests instead of regular Vue object?

Reply:
It would be best if we have a plugin that registers things like that. It's pretty handy when others use the library, or they'll need to make this loop as well ;)

Reply:
Oops, I didn't see these comments before merging. Andrey you bring up a good point, and it made me realize that registering directives in the install method will prevent people from using Recomponents piecemeal. 

Maybe components should import and register directives locally. That would remove the need for a plugin, or this setup loop for the testing harness. What do you guys think?